examples/advanced/xgboost/histogram-based/README.md

```diff
@@ -59,6 +59,92 @@ overwrite the `xgb_train()` method.
 To use other dataset, can inherit the base class `XGBDataLoader` and
 implement the `load_data()` method.
 
+## GPU support
+By default, CPU based training is used.
+
+If the CUDA is installed on the site, tree construction and prediction can be
+accelerated using GPUs.
+
+To enable GPU accelerated training, in `config_fed_client` set the args of 
+`FedXGBHistogramExecutor` to `"use_gpus": true` and set `"tree_method": "hist"`
+in `xgb_params`.
+
+We can also use the `device` parameter to map each rank to a GPU device ordinal in `xgb_params`.
```

Comment:
Does this device parameter section also apply to the tree-based version?

Reply:
right now it does not, since we have not cleaned up the tree executor